### ISSUE TITLE
Remove elevation >= 0 filter from observations api

### ISSUE THREAD

**Issue by emiliom** [communitysnowobs/cso-api#21]
See SQL WHERE condition at https://github.com/communitysnowobs/cso-api/blob/master/src/observations/query.js#L17

Having that filter makes it impossible to obtain some of the data in the CSO database and therefore reduces the value of the CSO `observations` API for QA/QC'ing the CSO database. Also, it assumes the elevation data are perfect, and that no valid data could be associated with negative elevation values generated by the Google Elevation API.

**Comment by dfosterhill** [comment]
Emilio, we should have a Jan call where we talk about these types of issues. Thank you for looking into it…

---------------------------------------------------
David F. Hill
Professor
Civil and Construction Engineering
Oregon State University
220 Owen Hall
Corvallis, OR 97331
541.737.4939
david.hill@oregonstate.edu<mailto:david.hill@oregonstate.edu>
http://www.engr.oregonstate.edu/~dfh
---------------------------------------------------

On Dec 24, 2020, at 2:44 PM, Emilio Mayorga <notifications@github.com<mailto:notifications@github.com>> wrote:


[This email originated from outside of OSU. Use caution with links and attachments.]


See SQL WHERE condition at https://github.com/communitysnowobs/cso-api/blob/master/src/observations/query.js#L17

Having that filter makes it impossible to obtain some of the data in the CSO database and therefore reduces the value of the CSO observations API for QA/QC'ing the CSO database. Also, it assumes the elevation data are perfect, and that no valid data could be associated with negative elevation values generated by the Google Elevation API.

**Comment by emiliom** [comment]
Right. I'm going to start compiling the discussions we've had recently about the API's, the CSO database, and so on, to facilitate our discussions on a future call.
